rtv/page.py

```diff
@@ -369,11 +416,16 @@ def _add_cursor(self):
     def _remove_cursor(self):
         self._edit_cursor(curses.A_NORMAL)
 
-    def _move_cursor(self, direction):
+    def _move_cursor(self, direction, page_ud=False):
 
         self._remove_cursor()
 
-        valid, redraw = self.nav.move(direction, len(self._subwindows))
+        if page_ud:
+            valid, redraw = self.nav.move_page(direction,
+                                               len(self._subwindows))
```

Comment:
I think using `len(self._subwindows)` is the wrong thing to do here. Here's an example:

Say the screen looks like this with '>' being the cursor.
```
  post 9
  post 10
  post 11
  post 12
  post 13
> post 14
  post 15
```

When you page up, the screen will move up 6 posts.

```
  post 3
  post 4
  post 5
  post 6
  post 7
> post 8
  post 9
```

Now consider what happens if `post 5` is realllly large and takes up 4 times the amount of room on the screen. After paging up, your screen will look like this
```
  post 3
  post 4
  post 5




> post 6
```
This is a big problem because we accidentally skipped right over `post 7` and `post 8`! Paging down will have a similar problem if `nav.inverted == True` because in that case, the page is relative to the bottom of the screen. Does that make sense?